common/list/lists-en.html

```diff
@@ -0,0 +1,583 @@
+---
+{
+	"title": "GCWeb Lists example",
+	"language": "en",
+	"altLangPage": "lists-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-03-05"
+}
+---
+
+<h1> Lists</h1>
```

Comment:
With the generic default page layout, you don't need to specify the heading level 1 on the page. It will be added by using the value set on the front-matter page properties "title"

```suggestion
```